packages/customer-account-ui-extensions/src/extension-points/extension-points.ts

```diff
@@ -39,24 +40,24 @@ export interface ExtensionPoints {
     StandardApi<'customer-account.order-index.block.render'>,
     AllComponents
   >;
-  'customer-account.account-information.block.render': RenderExtension<
-    StandardApi<'customer-account.account-information.block.render'>,
+  'customer-account.profile.block.render': RenderExtension<
+    StandardApi<'customer-account.profile.block.render'>,
     AllComponents
   >;
-  'customer-account.account-information.company-details.render-after': RenderExtension<
-    StandardApi<'customer-account.account-information.company-details.render-after'>,
+  'customer-account.profile.company-details.render-after': RenderExtension<
+    StandardApi<'customer-account.profile.company-details.render-after'>,
     AllComponents
   >;
-  'customer-account.account-information.addresses.render-after': RenderExtension<
-    StandardApi<'customer-account.account-information.addresses.render-after'>,
+  'customer-account.profile.addresses.render-after': RenderExtension<
+    StandardApi<'customer-account.profile.addresses.render-after'>,
     AllComponents
   >;
-  'customer-account.account-information.payment.render-after': RenderExtension<
-    StandardApi<'customer-account.account-information.payment.render-after'>,
+  'customer-account.profile.payment.render-after': RenderExtension<
+    StandardApi<'customer-account.profile.payment.render-after'>,
     AllComponents
   >;
-  'customer-account.account-information.location.render-after': RenderExtension<
-    StandardApi<'customer-account.account-information.location.render-after'>,
+  'customer-account.profile.location.render-after': RenderExtension<
+    StandardApi<'customer-account.profile.location.render-after'>,
```

Comment:
The [B2B Figma](https://www.figma.com/file/5Op17XqUmqj8dCSI5RZvo6/Extensibility-Dev-Preview?type=design&node-id=1656-12268&mode=design&t=gAHRJyzRBkBalmoZ-0) has `...staff-and-permission.render-after` now. Did that replace this one? cc @joao-quintino